deepchem/feat/materials_featurizers.py

```diff
@@ -0,0 +1,250 @@
+"""
+Featurizers for inorganic crystals.
+"""
+
+import numpy as np
+
+from deepchem.feat import Featurizer
+from deepchem.utils import pad_array
+
+
+class ChemicalFingerprint(Featurizer):
+  """
+  Chemical fingerprint of elemental properties from composition.
+
+  Based on the data source chosen, returns properties and statistics
+  (min, max, range, mean, standard deviation, mode) for a compound
+  based on elemental stoichiometry. E.g., the average electronegativity
+  of atoms in a crystal structure. The chemical fingerprint is a 
+  vector of these statistics. For a full list of properties and statistics,
+  see ElementProperty(data_source).feature_labels().
```

Comment:
Can you add a reference that you mean from `matminer.featurizers.composition import ElementProperty` and put the code in brackets for correct rendering?